File: src/main/java/com/example/solidaryaction/entities/Usuario.java
package com.example.solidaryaction.entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Usuario {

    public Long id;

    public String nome;

    public String cpf;

    public String senha;

    public String email;


}

File: front/app/types/usuario.ts
export class Usuario{
    constructor(
        public id: number | null,
        public nome:string,
        public email:string,
        public status:string,
        public cpf:string,
        public senha:string
    ){}
}
// prova - criar objeto
export interface UsuarioFormProps{
    usuarioExistente?:Usuario
}